test-framework/org.jboss.tools.openshift.reddeer/src/org/jboss/tools/openshift/reddeer/utils/TestUtils.java

```diff
@@ -142,6 +148,41 @@ public static String getValueOrDefault(String value, String defaultValue) {
 		}
 		return value;
 	}
+	
+	/**
+	 * Provide resource absolute path in project directory
+	 * @param path - resource relative path
+	 * @return resource absolute path
+	 */
+	public static String getProjectAbsolutePath(String... path) {
+
+		// Construct path
+		StringBuilder builder = new StringBuilder();
+		for (String fragment : path) {
+			builder.append("/" + fragment);
+		}
+
+		String filePath = "";
+		filePath = System.getProperty("user.dir");
```

Comment:
why declare and initialise with an empty string that immediately gets overwritten by the system property?